### PR TITLE
Incrementalified

### PR DESCRIPTION
**Pull request by jglick**

### CODE REVIEW

**Comment by jglick** [comment]
Mirror outage, retrying.

File: src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/StatusColumn.java [hunk 1]
```diff
@@ -23,7 +22,7 @@ public FlowNodeViewColumn getDefaultInstance() {
 
         @Override
         public String getDisplayName() {
-            return Messages.StatusColumn_DisplayName();
+            return "Status";
```
Comment (carlossg) [inline src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/StatusColumn.java]: unrelated?

Comment (jglick) [reply src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/StatusColumn.java]: No, this is required in order for the plugin to be buildable after updating the parent POM.
